api/specs/webserver/openapi-auth.yaml

```diff
@@ -657,7 +657,7 @@ components:
         name:
           title: Name
           type: string
-          description: Code name to the front-end page
+          description: Code name to the front-end page. Ideally a PageStr
```

Comment:
I am not sure I understand that change..

Reply:
I did not do that change. It happened because the attached model doc was changed and the OAS was not run. I am thinking that we might want to add an CI job to produce OAS instead of doing it manually